ocaml/xapi/certificates.ml

```diff
@@ -342,17 +345,21 @@ let host_install kind ~name ~cert =
       (ExnHelper.string_of_exn e) ;
     raise_library_corrupt ()
 
-let host_uninstall kind ~name =
+let host_uninstall ?(force = false) kind ~name =
```

Comment:
```suggestion
let host_uninstall kind ~name ~force =
```
There's no need to make the parameter optional. Optional parameters complicate usage in some occasions, because of their interaction with currying; and they are really only useful in situations where compatibility needs to be maintained, or other contrived situations.

In cases where we control all the users, such as this one, it's better to use named arguments